app/controllers/admin/people_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Admin
```

Comment:
Mmmmmm if I wrote those specs once I added the view then this commit would be under 100% coverage. I am aiming to have every single one of my commits at 100% coverage.

The alternative is to have the view commit fixup'd into this one but I feel like that's too much work for one commit.